control-plane/csi-driver/src/context.rs

```diff
@@ -33,6 +42,13 @@ pub enum Parameters {
     FileSystem,
     #[strum(serialize = "protocol")]
     ShareProtocol,
+    // TODO: Get the values from constant.
+    #[strum(serialize = "csi.storage.k8s.io/pvc/name")]
+    PvcName,
+    #[strum(serialize = "csi.storage.k8s.io/pvc/namespace")]
+    PvcNamespace,
+    #[strum(serialize = "openebs.io/volumegroup")]
```

Comment:
Does this mean we'd have to put "openebs.io/volumegroup" in the storage class?
If so I might suggest simply `volumeGroup` which is in keeping with the other parameters there.